programs/programs/policyengine/calculators/dependencies/spm.py

```diff
@@ -105,6 +105,13 @@ def value(self):
         return self.screen.has_expense(["telephone"])
 
 
+class HasUsdaElderlyDisabledDependency(SpmUnit):
+    field = "has_usda_elderly_disabled"
+
+    def value(self):
+        return any(member.age >= 60 or member.has_disability() for member in self.members)
+
+
```

Comment:
You can remove this and replace it with the member age dependency because PE will use the age variable, and the disabled variable to calculate this.